### PR TITLE
Fix import-sra to work with input sanitization fix and add test

### PR DESCRIPTION
**Pull request by mzganec**
Caching option in `fastq-dump` was changed from disabled to enabled which speeds up download of reads.

### CODE REVIEW

File: resolwe_bio/processes/import_data/sra_file.yml [hunk 1]
```diff
@@ -156,26 +156,33 @@
     runtime: polyglot
     language: bash
     program: |
-
+      CACHE=$PWD/fastq_cache
```
Comment (kostko) [inline resolwe_bio/processes/import_data/sra_file.yml]: This is incorrect as `$PWD` may contain spaces. Please quote the string in double quotes.

File: resolwe_bio/processes/import_data/sra_file.yml [hunk 2]
```diff
@@ -156,26 +156,33 @@
...
+      mkdir $CACHE
```
Comment (kostko) [inline resolwe_bio/processes/import_data/sra_file.yml]: Quote the argument in double quotes.

File: resolwe_bio/processes/import_data/sra_file.yml [hunk 3]
```diff
@@ -156,26 +156,33 @@
...
       mkdir $HOME/.ncbi/
```
Comment (kostko) [inline resolwe_bio/processes/import_data/sra_file.yml]: Quote the argument in double quotes.

File: resolwe_bio/processes/import_data/sra_file.yml [hunk 4]
```diff
@@ -287,15 +294,25 @@
     runtime: polyglot
     language: bash
     program: |
-
+      CACHE=$PWD/fastq_cache
```
Comment (kostko) [inline resolwe_bio/processes/import_data/sra_file.yml]: Same as above.

**Comment by mstajdohar** [comment]
Tests fail

**Comment by mzganec** [comment]
@mstajdohar Tests fail because of an error on Jenkins that needs to be fixed by someone with sufficient credentials. Tests fail early, before SRA import test even starts (see console output).

**Comment by mstajdohar** [comment]
cc @tjanez

**Comment by tjanez** [comment]
@mzganec, @mstajdohar, fixed.

**Pull request merged**